transformation/src/test/test-files/inputs/AntibodyLot/part1.json

```diff
@@ -0,0 +1,4 @@
+{"@id":"/antibodies/ENCAB697XQW/","@type":["AntibodyLot","Item"],"accession":"ENCAB697XQW","aliases":[],"alternate_accessions":[],"antigen_description":"Recognizes the FLAG peptide sequence:  DYKDDDDK","award":"/awards/U54HG006998/","characterizations":["/antibody-characterizations/68ece28d-1b38-4080-abe3-9ab0ea65eb2e/","/antibody-characterizations/e5467c14-712d-4243-a62d-2a1aa256efba/"],"clonality":"monoclonal","date_created":"2015-12-16T18:10:57.886604+00:00","dbxrefs":["AR:AB_262044"],"host_organism":"/organisms/mouse/","isotype":"IgG1","lab":"/labs/richard-myers/","lot_id":"SLBK1346V","lot_id_alias":[],"lot_reviews":[{"organisms":["/organisms/human/"],"detail":"Fully characterized.","biosample_term_id":"EFO:0001187","biosample_term_name":"HepG2","status":"characterized to standards","targets":["/targets/FLAG-synthetic_tag/"]},{"organisms":["/organisms/human/"],"detail":"Fully characterized.","biosample_term_id":"EFO:0001203","biosample_term_name":"MCF-7","status":"characterized to standards","targets":["/targets/FLAG-synthetic_tag/"]},{"organisms":["/organisms/human/"],"detail":"Fully characterized.","biosample_term_id":"EFO:0002067","biosample_term_name":"K562","status":"characterized to standards","targets":["/targets/FLAG-synthetic_tag/"]}],"product_id":"F1804","purifications":["affinity"],"schema_version":"9","source":"/sources/sigma/","status":"released","submitted_by":"/users/28bb3427-86e6-480a-be97-922c8b489f6a/","targets":["/targets/FLAG-synthetic_tag/","/targets/3xFLAG-synthetic_tag/"],"title":"ENCAB697XQW","url":"http://www.sigmaaldrich.com/catalog/product/sigma/f1804?lang=en&region=US","used_by_biosample_characterizations":[],"uuid":"534171de-26f6-4363-8a1e-a353357daea9"}
```

Comment:
(Commenting on random line) Can we add a test case w/ multiple values in the targets array?